torch/distributed/pipeline/sync/microbatch.py

```diff
@@ -40,9 +40,11 @@ class Batch:
 
     """
 
-    def __init__(self, value: TensorOrTensors) -> None:
+    def __init__(self, value: TensorOrTensors, is_single_sequence=False) -> None:
         self.value = value
         self.atomic = torch.is_tensor(value)
+        # For backward compatibility.
+        self.is_single_sequence = is_single_sequence
```

Comment:
how `is_single_sequence` is different from `atomic`?

Reply:
Sounds good. Maybe it would be better if you add a description of `is_single_sequence` to the docstring